include/etiss/dox/etiss_cfg.dox

```diff
@@ -10,9 +10,9 @@ The following table contains a list of current options. Plugins may have theire
 	    <th><b>Description</b></th>
 	  </tr>
 	  </tr><tr>
-	    <td> loglevel </td>
+	    <td> etiss.loglevel </td>
 	    <td> SILENT = 0 <br>FATALERROR = 1 <br>ERROR = 2 <br>WARNING = 3 (default) <br>INFO = 4 <br>VERBOSE = 5 <br><br></td>
-	    <td> sets the log level an suppresses messages of higher log levels.<br>e.g. "-floglevel=5" or "-ologlevel 5" </td>
+	    <td> sets the log level an suppresses messages of higher log levels.<br>e.g. "-fetiss.loglevel=5" or "-oetiss.loglevel 5" </td>
```

Comment:
this whole documentation page looks pretty outdated and incomplete. could you please update it? i don't think we should describe every single one here again, but instead refer to the "--help" command. Maybe the most important one to get started with ETISS make sense however. I think now that would only be the elf file

Reply:
I have updated this file.